Index: build/common.gypi
diff --git a/build/common.gypi b/build/common.gypi
index 780f43d2bbc7084f7d28517b711837bc43cd383a..a1af969cb942f519b745745e39a2b6ded4650fa3 100644
--- a/build/common.gypi
+++ b/build/common.gypi
@@ -2299,7 +2299,6 @@
         'release_valgrind_build': 1,
         'werror': '',
         'component': 'static_library',
-        'use_system_zlib': 0,
       }],
 
       # Build tweaks for DrMemory.
Index: build/linux/unbundle/zlib.gyp
diff --git a/build/linux/unbundle/zlib.gyp b/build/linux/unbundle/zlib.gyp
index 0a85ff02eeef414fd09a2008e05984cffca87962..ec80d4aeb4dfae9b86c7733482ee481ce319ce58 100644
--- a/build/linux/unbundle/zlib.gyp
+++ b/build/linux/unbundle/zlib.gyp
@@ -44,24 +44,5 @@
         ],
       },
     },
-    {
-      'target_name': 'zip',
-      'type': 'static_library',
-      'dependencies': [
-        'minizip',
-        '../../base/base.gyp:base',
-      ],
-      'include_dirs': [
-        '../..',
-      ],
-      'sources': [
-        'google/zip.cc',
-        'google/zip.h',
-        'google/zip_internal.cc',
-        'google/zip_internal.h',
-        'google/zip_reader.cc',
-        'google/zip_reader.h',
-      ],
-    },
   ],
 }
Index: net/spdy/spdy_framer.cc
diff --git a/net/spdy/spdy_framer.cc b/net/spdy/spdy_framer.cc
index faf49b407b1ab329616e1019d30d3ce49724b220..de41774a0fdb83343dfd13dafc270120beced275 100644
--- a/net/spdy/spdy_framer.cc
+++ b/net/spdy/spdy_framer.cc
@@ -39,6 +39,7 @@ uLong CalculateDictionaryId(const char* dictionary,
                  dictionary_size);
 }
 
+#if !defined(USE_SYSTEM_ZLIB)
 // Check to see if the name and value of a cookie are both empty.
 bool IsCookieEmpty(const base::StringPiece& cookie) {
   if (cookie.size() == 0) {
@@ -57,6 +58,7 @@ bool IsCookieEmpty(const base::StringPiece& cookie) {
   }
   return (pos == 0) && ((cookie.size() - value_start) == 0);
 }
+#endif  // !defined(USE_SYSTEM_ZLIB)
 
 // Pack parent stream ID and exclusive flag into the format used by HTTP/2
 // headers and priority frames.
