--- a/script/bootstrap.py
+++ b/script/bootstrap.py
@@ -50,9 +50,9 @@
     libcc_static_library_path = os.path.join(dist_dir, 'static_library')
 
   if PLATFORM != 'win32':
-    # Download prebuilt clang binaries.
-    update_clang()
     if not args.disable_clang and args.clang_dir == '':
+      # Download prebuilt clang binaries.
+      update_clang()
       # Build with prebuilt clang.
       set_clang_env(os.environ)
 
