diff --git a/atom.gyp b/atom.gyp
index eb4302d..3d3e161 100644
--- a/atom.gyp
+++ b/atom.gyp
@@ -202,7 +202,6 @@
               'files': [
                 '<@(copied_libraries)',
                 '<(libchromiumcontent_dir)/locales',
-                '<(libchromiumcontent_dir)/icudtl.dat',
                 '<(libchromiumcontent_dir)/content_shell.pak',
                 '<(libchromiumcontent_dir)/natives_blob.bin',
                 '<(libchromiumcontent_dir)/snapshot_blob.bin',
diff --git a/atom/renderer/api/atom_api_spell_check_client.cc b/atom/renderer/api/atom_api_spell_check_client.cc
index 25d1e30..0657766 100644
--- a/atom/renderer/api/atom_api_spell_check_client.cc
+++ b/atom/renderer/api/atom_api_spell_check_client.cc
@@ -11,7 +11,7 @@
 #include "base/logging.h"
 #include "native_mate/converter.h"
 #include "native_mate/dictionary.h"
-#include "third_party/icu/source/common/unicode/uscript.h"
+#include <unicode/uscript.h>
 #include "third_party/WebKit/public/web/WebTextCheckingCompletion.h"
 #include "third_party/WebKit/public/web/WebTextCheckingResult.h"
 
diff --git a/chromium_src/chrome/renderer/spellchecker/spellcheck_worditerator.cc b/chromium_src/chrome/renderer/spellchecker/spellcheck_worditerator.cc
index 815a9c0..5a0d7f6 100644
--- a/chromium_src/chrome/renderer/spellchecker/spellcheck_worditerator.cc
+++ b/chromium_src/chrome/renderer/spellchecker/spellcheck_worditerator.cc
@@ -14,10 +14,10 @@
 #include "base/logging.h"
 #include "base/strings/stringprintf.h"
 #include "base/strings/utf_string_conversions.h"
-#include "third_party/icu/source/common/unicode/normlzr.h"
-#include "third_party/icu/source/common/unicode/schriter.h"
-#include "third_party/icu/source/common/unicode/uscript.h"
-#include "third_party/icu/source/i18n/unicode/ulocdata.h"
+#include <unicode/normlzr.h>
+#include <unicode/schriter.h>
+#include <unicode/uscript.h>
+#include <unicode/ulocdata.h>
 
 // SpellcheckCharAttribute implementation:
 
diff --git a/chromium_src/chrome/renderer/spellchecker/spellcheck_worditerator.h b/chromium_src/chrome/renderer/spellchecker/spellcheck_worditerator.h
index 2ac28a2..0f88da6 100644
--- a/chromium_src/chrome/renderer/spellchecker/spellcheck_worditerator.h
+++ b/chromium_src/chrome/renderer/spellchecker/spellcheck_worditerator.h
@@ -14,7 +14,7 @@
 #include "base/basictypes.h"
 #include "base/memory/scoped_ptr.h"
 #include "base/strings/string16.h"
-#include "third_party/icu/source/common/unicode/uscript.h"
+#include <unicode/uscript.h>
 
 namespace base {
 namespace i18n {
